packages/backend/src/nest/socket/socket.service.ts

```diff
@@ -44,12 +46,15 @@ export class SocketService extends EventEmitter implements OnModuleInit {
     this.readyness = new Promise<void>(resolve => {
       this.resolveReadyness = resolve
     })
+
+    this.sockets = new Set<net.Socket>()
+
+    this.attachListeners()
```

Comment:
[FYI] attachListeners() moved from `onModuleInit()` to the `constructor`. I'm not sure what side effects this would have. Might cause `attachListeners` to trigger before `init`?

Reply:
Yea it does trigger before `init` and I think that's fine. I think it makes sense to put everything necessary to setup an object in the constructor and only using `onModuleInit` when necessary and it doesn't seem like `attachListeners` needs to be in `onModuleInit`.